docs/src/components/ColorPalette.js

```diff
@@ -0,0 +1,43 @@
+// @flow strict
+import type { Node } from 'react';
+import { Box, Text } from 'gestalt';
+// $FlowFixMe[untyped-import]
+import tokens from 'gestalt-design-tokens/dist/json/variables.json';
+
+type Props = {|
+  name: string,
+  tokenId: string,
+|};
+
+const ColorPalette = ({ name, tokenId }: Props): Node => {
+  const tokenNumbers = [0, 50, 100, 200, 300, 400, 500, 600, 700, 800, 900];
+  const colorBoxes = tokenNumbers.map((number, idx) => {
```

Comment:
nit: this isn't reused, so it should either be inlined into the main markup below, or pulled out to a separate child component